hooks/pre/30-upgrade.rb

```diff
@@ -60,15 +68,20 @@ def fail_and_exit(message)
   upgrade_step :stop_services
   upgrade_step :start_databases
 
-  if Kafo::Helpers.module_enabled?(@kafo, 'katello') || @kafo.param('capsule', 'pulp').value
+  if Kafo::Helpers.module_enabled?(@kafo, 'katello') || @kafo.param('foreman_proxy_plugin_pulp', 'pulpnode_enabled').value
     upgrade_step :migrate_pulp
     upgrade_step :start_httpd
   end
 
+  if @kafo.param('foreman_proxy_plugin_pulp', 'pulpnode_enabled').value
+    upgrade_step :remove_nodes_importers
```

Comment:
@stbenjam I would think migrate_pulp and start_httpd should happen  on both a capsule and katello. If I move those to a block that only checks for katello, then they won't happen on a capsule as well?

This is how I read it now:
```
if capsule or katello
  migrate pulp
  start httpd

if capsule
  remove nodes importers

if katello
 migrate candlepin
 migrate foreman
 migrate gutterball
 remove nodes distributors